ext/sockets/conversions.c

```diff
@@ -720,6 +720,10 @@ static void from_zval_write_sockaddr_aux(const zval *container,
 		zend_llist_add_element(&ctx->keys, &node);
 		from_zval_write_int(elem, (char*)&family, ctx);
 		zend_llist_remove_tail(&ctx->keys);
+
+		if (UNEXPECTED(ctx->err.has_error)) {
+			return;
+		}
 	} else {
 		family = ctx->sock->type;
```

Comment:
Would moving this to be an unconditional set prior to the if branch get rid of the compiler warning?

Reply:
Sounds good to me :)